src/pip/_internal/wheel_builder.py

```diff
@@ -79,9 +80,18 @@ def _should_build(
 
     if not req.use_pep517 and not is_wheel_installed():
         # we don't build legacy requirements if wheel is not installed
-        logger.info(
-            "Using legacy setup.py install for %s, "
-            "since package 'wheel' is not installed.", req.name,
+        deprecated(
+            reason=(
+                "Cannot build a wheel for {} which do not use PEP 517 "
+                "since package 'wheel' is not installed. "
+                "pip will fall back to legacy 'setup.py install' for this "
+                "package.".format(req.name)
+            ),
+            replacement=(
+                "to install the 'wheel' package or enable --use-pep517"
+            ),
+            gone_in="20.3",
```

Comment:
We need 6 months before changing behaviors, according to the deprecation policy.